### ISSUE TITLE
fix netty issue when referring to remote rikai jar

### ISSUE THREAD

**Issue by changhiskhan** [eto-ai/rikai#57]

**Comment by eddyxu** [comment]
After purging `~/.ivy` and re-run `sbt publishLocal`, got this error.

```

	  https://dl.bintray.com/spark-packages/maven/org/xerial/sbt/sbt-sonatype/3.9.5/sbt-sonatype-3.9.5.jar

		::::::::::::::::::::::::::::::::::::::::::::::

		::          UNRESOLVED DEPENDENCIES         ::

		::::::::::::::::::::::::::::::::::::::::::::::

		:: org.xerial.sbt#sbt-sonatype;3.9.5: not found

		::::::::::::::::::::::::::::::::::::::::::::::



:: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
Exception in thread "main" java.lang.RuntimeException: [unresolved dependency: org.xerial.sbt#sbt-sonatype;3.9.5: not found]
	at org.apache.spark.deploy.SparkSubmitUtils$.resolveMavenCoordinates(SparkSubmit.scala:1389)
	at org.apache.spark.deploy.DependencyUtils$.resolveMavenDependencies(DependencyUtils.scala:54)
	at org.apache.spark.deploy.SparkSubmit.prepareSubmitEnvironment(SparkSubmit.scala:308)
	at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:871)
	at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:180)
	at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:203)
	at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:90)
	at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1007)
	at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1016)
	at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
Traceback (most recent call last):
  File "/home/lei/work/rikai/venv/lib/python3.8/site-packages/pyspark/python/pyspark/shell.py", line 38, in <module>
    SparkContext._ensure_initialized()
```

**Comment by eddyxu** [comment]
Hmm, identified this is introduced by 

```
"com.google.cloud" % "google-cloud-storage" % gcpVersion,
```

i will remove this for now.

**Comment by eddyxu** [comment]
Fixed in #60
